tests/test_integration_new_sync.py

```diff
@@ -28,9 +28,9 @@ def set_agent_config(monkeypatch, tmp_work_dir):
     monkeypatch.setattr("agent.config.USING_DUMMY_DATA_BACKEND", True)
     # Note that as we are running ehrql actions in this test, we need to set
     # the backend to a value that ehrql will accept
-    # RAP API stage 2: Use `emis` rather than `test` so that we can disable the
+    # RAP API stage 2: Use `test` rather than `test` so that we can disable the
```

Comment:
Probably want to just remove this comment?

Reply:
hah yes!